### ISSUE TITLE
Modularise and centralise breadcrumb

### ISSUE THREAD

**Issue by gburton** [gburton/Responsive-osCommerce#689]
Breadcrumbs are created on a per page basis which is not ideal.

Propose to create breadcrumb in one place and simply display the correct breadcrumb for the correct page (and vars).

**Comment by gburton** [comment]
Anyone wish to take this on?

**Comment by osc2nuke** [comment]
I not understand exactly the question/problem.
Can you give a little more insight in the situation/problem, and perhaps your idea of how it should be?

**Comment by osc2nuke** [comment]
I did that since the beginning.
I made the breadcrumb a separated CM.
There is a HUGE challenge in it.
Getting the Site's PageTitle ( they are all based on the other CM's) on the LEFT and the generated Breadcrumbs on the Right. (This is just a design FLAW my end or template makers end)
But beside that, just make the breadcrumb a separated CM.
The rest of the design is upon them.
( i have somewhere the the code for it, but it is just a CM module, i not believe it is worth the effort to provide you with such simplicity)
